### ISSUE TITLE
UIScrollToSelection suggestions

### ISSUE THREAD

**Issue by SimonDarksideJ** [Unity-UI-Extensions/com.unity.uiextensions#186]
<sup>Issue created by Justin Hardage as [Bitbucket Issue #&#8203;150](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/150) on 2017.07.01 06:47.</sup>
1) It seems like UIScrollToSelection methods UpdateHorizontalScrollPosition() and UpdateVerticalScrollPosition() assume anchors of `0, 1`. I added some code to lines 173 and 190 to compensate for other anchor values.

Example: 
```C#

float selectionPosition = -selection.anchoredPosition.y - (selection.rect.height * (1 - selection.pivot.y));
```

2) Given that ScrollToSelection is functionality that users mostly want in menus, I felt it more useful to use `Time.unscaledDeltaTime` instead of `Time.deltaTime`. Check lines 183 and 200.

Example: 
```C#

TargetScrollRect.verticalNormalizedPosition +=
                (offlimitsValue / LayoutListGroup.rect.height) * Time.unscaledDeltaTime * scrollSpeed;
```

Does this functionality already exist in some setting that I haven't seen yet?

Would it be useful to add these to a pull request?

**Comment by SimonDarksideJ** [comment]
<sup>On [2017.07.01 14:35](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/150#comment-37965487), @SimonDarksideJ commented:</sup>
Contributions are always welcome, I can either take your changes from here or you can submit a fresh PR.  Up to you.

Let me know.

**Comment by SimonDarksideJ** [comment]
<sup>On [2017.07.01 15:41](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/150#comment-37966019), Justin Hardage commented:</sup>
I tried the PR, but it takes me to an Access Denied page. I also don't have access to push my branch to BitBucket. Since I apparently don't understand how Atlassian does PRs, feel free to snag my code from here. :D Thanks!

**Comment by SimonDarksideJ** [comment]
<sup>On [2017.07.01 18:19](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/150#comment-37967055), @SimonDarksideJ commented:</sup>
I've had several PR's from other devs, so not sure what the problem is. Normally people fork, then commit changes to a new branch and then submit a PR back to the main repo with that branch.
But I'll grab the code nonetheless. :D

**Comment by SimonDarksideJ** [comment]
<sup>On [2017.07.02 14:44](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/150#comment-37972386), @SimonDarksideJ commented:</sup>
Adopts changes proposed in #150  ScrollTo now works with Pivot

Resolves #150


Referenced by 50c41198db48

**Comment by SimonDarksideJ** [comment]
<sup>On 2017.07.02 14:44 @SimonDarksideJ modified issue:</sup>
**status** changed `new` → `resolved`
